apps/src/sites/studio/pages/java_ide/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import JavaIdeView from '../../../../javaide/JavaIdeView';
+import {getStore} from '../../../../code-studio/redux';
```

Comment:
NIT: A shorthand for these imports is "@cdo/apps/code-studio/redux"

Reply:
fixed!